pages/index.js

```diff
@@ -33,26 +74,25 @@ export default function HomePage() {
   if (!data) {
     return <p>loading</p>;
```

Comment:
Im not sure but if data is not true, shouldnt there be an error message instead of a loading message?